airflow/www_rbac/templates/airflow/conn_create.html

```diff
@@ -19,5 +19,5 @@
 
 {% block tail %}
   {{ super() }}
-    <script src="{{ url_for('static', filename='connection_form.js') }}"></script>
+    <script src="{{ url_for_asset('connectionForm.js') }}"></script>
```

Comment:
Why the change of underscore to camelCase? There doesn't seem to be a connectionForm.j either?

Reply:
https://github.com/apache/incubator-airflow/pull/3572#discussion-diff-200991459R40